src/components/APIDoc/SchemaDataView.tsx

```diff
@@ -56,15 +61,80 @@ const PropertyView:React.FunctionComponent<PropertyComponentProps> = ({propName,
       </FlexItem>
       <FlexItem>
         <TextContent>
-            <Text component={TextVariants.p}>
-              {propertyType}
-            </Text>
-          </TextContent>
+          <Text component={TextVariants.p}>
+            {propertyType}
+          </Text>
+        </TextContent>
+      </FlexItem>
+      <FlexItem>
+        {extraProps}
       </FlexItem>
     </Flex>
   )
 }
 
+
+interface ExtraPropertyViewProps {
+  propSchema: OpenAPIV3.SchemaObject;
+}
+const ExtraPropertyView:React.FunctionComponent<ExtraPropertyViewProps> = ({propSchema}) => {
```

Comment:
We could move it to a separate file, but we could do that later.

Reply:
I ended up moving the whole PropertyView, including ExtraPropertyView to a separate file.